starknet_py/tests/e2e/fixtures/abi_v1_structures.py

```diff
@@ -0,0 +1,75 @@
+# data from cairo repository: crates/cairo-lang-starknet/src/abi_test.rs
+from collections import OrderedDict
+
+from starknet_py.abi.v1.model import Abi
+from starknet_py.cairo.data_types import FeltType, StructType, UintType
+
+my_struct_dict = {
+    "type": "struct",
+    "name": "test::MyStruct::<core::integer::u256>",
+    "members": [
+        {"name": "a", "type": "core::integer::u256"},
+        {"name": "b", "type": "core::felt252"},
+    ],
+}
+my_struct = StructType(
+    name="test::MyStruct::<core::integer::u256>",
+    types=OrderedDict(a=UintType(256), b=FeltType()),
+)
+
+my_enum_dict = {
+    "type": "enum",
+    "name": "test::MyEnum::<core::integer::u128>",
+    "variants": [
+        {"name": "a", "type": "core::integer::u256"},
+        {"name": "b", "type": "test::MyStruct::<core::integer::u128>"},
+    ],
+}
+my_enum = StructType(
```

Comment:
They are built the same way as structs.
Also, it is much easier to keep them as structs since structs and enums can have each other inside (the logic for that is already implemented in the structure type).

Reply:
So why are we even defining `EnumType` in the first place? I'd not mix the logic for these two even if they are similiar.